### PR TITLE
end-to-end-age-model and updates

### PR DESCRIPTION
**Pull request by maxklema**
This pull request introduces the end-to-end framework for the facial recognition model.

Currently, we use ResNetV2_50 as a backbone and train the last 40 layers in addition to our head, which consists of three dense layers and two intermediate dropout layers. 

From testing, the MAE is around 5.7 years. The largest MAEs fall into older age groups (60+), while the shortest MAEs (4-5) fall into younger age groups (i.e., 10-20, 20-30, etc.). 

One thing to note is that all of the training data, or at least a good proportion of it, are cinemotography stills. This means that the model is trained on certain camera angles, lighting, filters, etc. When 'normal' photos are passed into the model, the model tends to perform worse. Meanwhile, stills from movies tend to be more accurate. Fortunately, the whole goal of the project is to take stills from films or TV shows. So, this difference should not be an issue later on.

### CODE REVIEW

**Review by Dillstan — APPROVED** [review]
Need to merge in the conflicts but otherwise looks good

**Review by IvieXue — APPROVED** [review]
